### ISSUE TITLE
How to get headers from $client->request()?

### ISSUE THREAD

**Issue by syropian** [artdarek/oauth-4-laravel#82]
The GitHub API paginates it's data and they include pagination info in the request headers. When calling `request()`, is there a way to access the headers from the response?

**Comment by dskanth** [comment]
Yes, i am also looking for a way to read the response headers. Does anyone have an example ?
